src/main/webapp/app/exercises/shared/result/result-detail.component.ts

```diff
@@ -191,13 +215,15 @@ export class ResultDetailComponent implements OnInit {
     private createFeedbackItems(feedbacks: Feedback[]): FeedbackItem[] {
         if (this.exerciseType === ExerciseType.PROGRAMMING) {
             return feedbacks.map((feedback) => {
+                const previewText = ResultDetailComponent.computeFeedbackPreviewText(feedback.detailText);
```

Comment:
```suggestion
                const previewText = this.computeFeedbackPreviewText(feedback.detailText);
```
Plus setting the method non-static.
For lambda ("arror") functions, the `this` reference is the same as the one from the defining scope.

Reply:
this does not work for me, at least IntelliJ gives a compile error.

Reply:
There actually is another occurrence at line 276. Then it should work.
IntelliJ suggest making it static because it doesn't reference any non-static method or property. Since we had some discussions previously regarding not making anything static unless necessary, I decided to mention it here. If you decide you want to follow the IntelliJ suggestion, that's okay for me too.

Reply:
Let's keep it static